src/components/shared/select/Select.tsx

```diff
@@ -0,0 +1,24 @@
+import "./Select.scss";
+
+export interface SelectProps {
+  selectName: string;
+  data: string[];
+}
+
+const Select = ({ selectName, data }: SelectProps) => {
+  return (
+    <div className="select">
+      <label>{selectName}</label>
+      <div>
+        <span></span>
+        <ul>
+          {data.map((singleData) => (
+            <li>{singleData}</li>
```

Comment:
there is probably error in the console :)

Reply:
fixed.